index.js

```diff
@@ -318,7 +318,9 @@ Tree.prototype._join = function (data) {
     }
   }
 
+  this.overflowClassed(data.length)
```

Comment:
I was using that at first actually! But then I thought about it from a calling standpoint. And it seems like that's a direction, "Hey tree, go resize yourself". But it's not resizing, it's just setting a class whether or not there is scrollableContent. Maybe `onresize` would make more sense?